CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+v3.1.0 / 2022-01-03
+========================
+* feat(api): adding v1 CRDs ([#167](https://github.com/openebs/jiva-operator/pull/167),[@shubham14bajpai](https://github.com/shubham14bajpai))
+* refactor(operator): use a common label (jivaapi) to reference jiva api ([#171](https://github.com/openebs/jiva-operator/pull/171),[@abhisheksinghbaghel](https://github.com/abhisheksinghbaghel))
+* fix(provisioning): corrected the prometheus scrape annotation syntax ([#174](https://github.com/openebs/jiva-operator/pull/174),[@shazadbrohi](https://github.com/shazadbrohi))
+* fix(helm): Make log verbosity configurable ([#161](https://github.com/openebs/jiva-operator/pull/161),[@ianroberts](https://github.com/ianroberts))
```

Comment:
Do we also need to mention #165 ?

Reply:
No it is just a fix on this so not required